### PR TITLE
net-mail/automx2: Bump to version 2022.0

### PR DESCRIPTION
**Pull request by rseichter**
Upstream feature release. This ebuild adds Python 3.10 to PYTHON_COMPAT.

### CODE REVIEW

File: net-mail/automx2/automx2-2022.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9,10} )
```
Comment (juippis) [inline net-mail/automx2/automx2-2022.0.ebuild:6]: Can we clean the unsupported implementations out?

File: net-mail/automx2/automx2-2022.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline net-mail/automx2/automx2-2022.0.ebuild:4]: 8? :)

File: net-mail/automx2/automx2-2022.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://automx.org/"
+SRC_URI="https://github.com/rseichter/automx2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="acct-user/automx2"
```
Comment (juippis) [inline net-mail/automx2/automx2-2022.0.ebuild:18]: It doesn't look like `fowners` or `chown` is called in this ebuild, so this should be in RDEPEND I reckon.

**Review by juippis — COMMENTED** [review]
I can make the suggested changes and merge if you wish, let me know if you agree to them.

File: net-mail/automx2/automx2-2022.0.ebuild (hunk 1 above)
Comment (rseichter) [reply net-mail/automx2/automx2-2022.0.ebuild:6]: Why not. The above list is just what I use for [CI testing](https://github.com/rseichter/automx2/actions/runs/2070824930).

File: net-mail/automx2/automx2-2022.0.ebuild (hunk 2 above)
Comment (rseichter) [reply net-mail/automx2/automx2-2022.0.ebuild:4]: I haven't yet written any ebuild with V8. However, if you think it works, feel free to bump the version.

File: net-mail/automx2/automx2-2022.0.ebuild (hunk 3 above)
Comment (rseichter) [reply net-mail/automx2/automx2-2022.0.ebuild:18]: I thought that acct-user/* was always used during build time, but if you wish to move it to RDEPEND, I have no objections.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**